src/config/dynamic-theme-fixes.config

```diff
@@ -9083,6 +9083,18 @@ body
 
 ================================
 
+hm.com
+
+CSS
+.review-answers .review-answer .average-score {
+    background-color: var(--darkreader-neutral-text) !important;
+}
+.TurnToReviewsButton-module--ratingStars__gFrXo {
```

Comment:
What about using:

```css
INVERT 
[class="TurnToReviewsButton-module--ratingStars"]
```
?

Reply:
I didn't know that was a thing, will correct it

Reply:
fixed